packages/e2e-test-utils-playwright/src/page/index.ts

```diff
@@ -50,6 +52,10 @@ class PageUtils {
 	openPreviewPage = openPreviewPage;
 	setBrowserViewport = setBrowserViewport;
 	pressKeyTimes = pressKeyTimes;
+	getCurrentUser = getCurrentUser;
+	switchUserToAdmin = switchUserToAdmin;
+	switchUserToTest = switchUserToTest;
```

Comment:
It looks like these three utils are now unused since the `activatePlugin` and `deactivatePlugin` utils were removed.

What do you think about removing these too until they're needed?

Reply:
Yeah you are right, removed these